Lesson-6/assignment/src/components/Employee.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react'
+import { Card, Col, Row, Layout, Alert, message, Button } from 'antd';
+
+import Common from './Common';
+
+class Employee extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {};
+    }
+
+    componentDidMount() {
+        this.checkEmployee();
```

Comment:
you should register events like in common.js here e.g.`this.getPaidEvent = payroll.GetPaid(updateInfo);` and stop watch in `componentWillUnmount`

you could think about what events to listen here